foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -27,9 +33,8 @@
 MAX_PROCESSOR_RUN_TIME = timedelta(days=1)
 MAX_QUEUE_TIME = timedelta(days=1)
```

Comment:
Seems too high and too low, repsectively. This should mayyyybe be variablized? Although then it's more fine tuning to do.